### ISSUE TITLE
Direct Time Entry

### ISSUE THREAD

**Issue by danogeek** [jason0x43/alfred-toggl#63]
I don't use timers, but just enter time directly against projects. Is there any way to do that with this Alfred workflow?

I enter a description, pick a project, sometimes add a tag, then enter the time. Time, decimal, hours, or minutes format would all work - I don't care which.

**Comment by jason0x43** [comment]
I don't believe so, but it's been quite some time since I've used this workflow.
